tests/structure_matching_tests.rs

```diff
@@ -9,8 +9,8 @@ fn test_exact_match() {
     let romol1 = ROMol::from_smiles(smiles1).unwrap();
     let romol2 = ROMol::from_smiles(smiles2).unwrap();
 
-    assert!(exact_match(&romol1, &romol1.clone()));
-    assert!(!exact_match(&romol1, &romol2));
+    assert!(exact_match(&romol1, &romol1.clone(), true));
+    assert!(!exact_match(&romol1, &romol2, true));
```

Comment:
Can you make a test where you explore what happens in a scenario when use-chirality is true AND when it's false?

Reply:
Good call! Went ahead and added a test to compare two molecules that match when use-chirality is set to false but fail to match when use-chirality is set to true.